transit_odp/site_admin/exports.py

```diff
@@ -232,13 +255,17 @@ class DatasetPublishingCSV(CSVBuilder):
                 round(ppc_overall_score(dataset.organisation_id))
             )
             + "%"
-            if dataset.percent_matching >= 0 and dataset.status != INACTIVE
+            if ppc_overall_score(dataset.organisation_id)
+            and dataset.percent_matching >= NO_PPC_DATA
+            and dataset.status != INACTIVE
             else "",
         ),
         CSVColumn(
             header="Archived matching reports URL",
             accessor=lambda dataset: get_ppc_weekly_overall_url(dataset.organisation_id)
-            if dataset.percent_matching >= 0 and dataset.status != INACTIVE
+            if ppc_overall_score(dataset.organisation_id)
```

Comment:
In what circumstances will `ppc_overall_Score` return a falsy value?

Reply:
`ppc_overall_score` return a float, all the `if` statement return:
 1. false if is INACTIVE 
 2. when "% AVL to Timetables feed matching score" is:
     - number
     - Undergoing ..
     -  Due to ....
     Has to be true

Reply:
The purpose of `ppc_overall_score` is create a `QuerySet`, it returns a `float` but most important it queries the db
calling `.add_avl_compliance_status_cached` in the `AVLDataset`, Unfortunately I can't call in `self.get_queryset` (Dataset).
Probably here it not really clear, but the aim of the `if` is not to see true or false but call the `QuerySet`
I mean I am using the `if` to call a function.
This is what I meant in the first commit message 👆

Reply:
"If" you think in the `if` that call is always `True`, "if" you think in the `function` there has been called and it has been called `.add_avl_compliance_status_cached`

Reply:
Yes, `ppc_overall_score` creates and executes a QuerySet - but it returns the average value and **discards** the rest of the QuerySet. It's not possible for the contents of this QuerySet to be used elsewhere.

Where do you think it's being used?

(If you really do want to retain values in the QuerySet, you would need to make it a member of the CSV builder class.)